mapzap/models/staging/base/base_dcp__pluto_with_bbls.sql

```diff
@@ -0,0 +1,32 @@
+{% set pluto_versions_query %}
```

Comment:
What's this line do? Where is it actually getting this query from?

Interesting file btw, cool to see the jinja iteration in action

Reply:
Gotcha. Didn't realize that was valid syntax, but also should have just read a little more closely